fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionFragment.java

```diff
@@ -149,14 +150,27 @@ public View onCreateView(
                   "User should not be able to use the action button in this state :" + state);
           }
         });
-
-    election.getState().observe(getViewLifecycleOwner(), eventState -> setupElectionContent());
+    laoDetailViewModel
+        .getCurrentLao()
+        .observe(
+            getActivity(),
```

Comment:
I think it is better to use `getViewLifecycleOwner()`. It is non-null and more suited to what we want.